core/primitives/src/version.rs

```diff
@@ -173,3 +177,42 @@ macro_rules! checked_feature {
         }
     }};
 }
+
+#[macro_export]
+macro_rules! checked_feature_crate {
+    ($feature_name:tt, $feature:ident, $current_protocol_version:expr) => {{
+        #[cfg(feature = $feature_name)]
+        let is_feature_enabled = crate::version::PROTOCOL_FEATURES_TO_VERSION_MAPPING
+            [&crate::version::ProtocolFeature::$feature]
```

Comment:
Is the only change here `near_primitives::` being renamed into `crate::`? What is the issue you have faced with the original macro?

Reply:
It has no `near_primitives::` and unable to compile.

Reply:
I see. We need to use `$crate` instead of `near_primitives`: https://doc.rust-lang.org/1.5.0/book/macros.html#the-variable-crate

Reply:
It seems $crate works perfectly. Thank you for the insight. Updated.